### ISSUE TITLE
Go through API Diff report and check for unwanted changes due Xtend 2 Java mig

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext#1733]
Go through API Diff report and check for unwanted changes due Xtend 2 Java mig

https://ci.eclipse.org/xtext/job/releng/job/xtext-xtend-api-diff

**Comment by cdietrich** [comment]
```

Status | Modifier | Type | Method
-- | -- | -- | --
REMOVED                 (!) | protected | org.eclipse.jdt.core.IPackageFragmentRoot | _getPackageFragmentRoot(org.eclipse.jdt.core.IJavaElement)
REMOVED                 (!) | protected | org.eclipse.jdt.core.IPackageFragmentRoot | _getPackageFragmentRoot(org.eclipse.jdt.core.IPackageFragmentRoot)
REMOVED                 (!) | protected | org.eclipse.jdt.core.IPackageFragmentRoot | _getPackageFragmentRoot(java.lang.Void)

```
@szarnekow do we want these changes?

**Comment by szarnekow** [comment]
I think this particular case is fine.

**Comment by miklossy** [comment]
I went through the API diff report and did not find any unwanted changes.

Closing as fixed in 2.22.
